home/video.asm

```diff
@@ -70,6 +70,11 @@ UpdateBGMapBuffer::
 	ld bc, wBGMapPalBuffer
 	ld de, wBGMapBuffer
 
+; We increment the low byte of a pointer, so ensure these buffers
+; dont cross a 256 byte boundary
+assert HIGH(wBGMapBuffer) == HIGH(wBGMapBuffer + 47)
```

Comment:
The 47s are suspicious. Should those be `wBGMapBufferEnd` and `wBGMapPalBufferEnd`?

Reply:
Yeah i should just add labels, would also prevent it from breaking if they change size